spacy/lang/nl/punctuation.py

```diff
@@ -0,0 +1,39 @@
+# coding: utf8
+from __future__ import unicode_literals
+
+from ..char_classes import LIST_ELLIPSES, LIST_ICONS
+from ..char_classes import CONCAT_QUOTES, ALPHA, ALPHA_LOWER, ALPHA_UPPER
+from ..punctuation import TOKENIZER_SUFFIXES as DEFAULT_TOKENIZER_SUFFIXES
+
+
+# Copied from `de` package. Main purpose is to ensure that hyphens are not
+# split on.
+_quotes = CONCAT_QUOTES.replace("'", "")
+
+_infixes = (
+    LIST_ELLIPSES
+    + LIST_ICONS
+    + [
+        r"(?<=[{al}])\.(?=[{au}])".format(al=ALPHA_LOWER, au=ALPHA_UPPER),
+        r"(?<=[{a}])[,!?](?=[{a}])".format(a=ALPHA),
+        r"(?<=[{a}])[:<>=](?=[{a}])".format(a=ALPHA),
+        r"(?<=[{a}]),(?=[{a}])".format(a=ALPHA),
+        r"(?<=[{a}])([{q}\)\]\(\[])(?=[{a}])".format(a=ALPHA, q=_quotes),
```

Comment:
IIRC, round brackets `()` are contained within the `CONCAT_QUOTES` var so could be removed from the regex here.

Reply:
Yeah, when I did the rebase to `develop`, I also just copied over the current rules from `de.punctuation` without really questioning them. So if it turns out there are improvements to be made here, we can also update the `de` rules accordingly 🎉

Reply:
@ines @lemontheme : It's nothing urgent or a big deal - so I'd be happy to look into this and check any other redundancies in other languages after this PR is merged! I did some initial cleanup during the regex PR but apparently didn't catch it all at the time ;-)